db/migrate/20150921143110_devise_create_users.rb

```diff
@@ -20,15 +20,15 @@ def change
       t.inet     :last_sign_in_ip
 
       ## Confirmable
-      # t.string   :confirmation_token
-      # t.datetime :confirmed_at
-      # t.datetime :confirmation_sent_at
-      # t.string   :unconfirmed_email # Only if using reconfirmable
+      t.string   :confirmation_token
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.